app/presenters/mfa_confirmation_show_presenter.rb

```diff
@@ -0,0 +1,28 @@
+class MfaConfirmationShowPresenter
+    include ActionView::Helpers::TranslationHelper
+    attr_reader :current_user, :final_path, :next_path
+    def initialize(current_user:, next_path:, final_path:)
+      @current_user = MfaContext.new(current_user)
```

Comment:
indentation seems to go in by 4 here instead of by 2, and the last `end` of the file doesn't seem to line with `class`?